### PR TITLE
Add PIM request documentation to azure docs

### PR DESCRIPTION
**Pull request by avinhurry**
Adding instructions to carry out PIM requests to the Azure docs.

### CODE REVIEW

File: source/infrastructure/hosting/azure-cip/index.html.md.erb [hunk 1]
```diff
@@ -34,6 +34,34 @@ The new user will receive an invitation by email. Then the service managers can
...
+
+* Click on `Azure resources` under `Activate`
+
+* You may have to lengthen the resource section in order to see the full resource name, including the environment
+
+* Click `Activate` under `Action` for the relevant resource
+
+* Select a duration (maximum 8 hours), enter a reason for the request and click `Activate`
+
+* An email will be sent to the administrators who can approve or deny the request
```
Comment (saliceti) [inline source/infrastructure/hosting/azure-cip/index.html.md.erb:61]: For the test subscription, approval is automatic and you should receive confirmation shortly.
For the production subscription, an email will be sent...

File: source/infrastructure/hosting/azure-cip/index.html.md.erb [hunk 2]
```diff
@@ -34,6 +34,34 @@ The new user will receive an invitation by email. Then the service managers can
 
 To access Azure Devops, the new user must access [Azure DevOps](https://dev.azure.com/dfe-ssp/). This will register them in the system then the service owner can add them to the project.
 
+## Privileged Identity Management (PIM) Requests
+
+Privileged Identity Management (PIM) is a service in Azure Active Directory (Azure AD) that enables you to manage, control, and monitor access to important resources at DfE such as access to staging and production environments.
```
Comment (saliceti) [inline source/infrastructure/hosting/azure-cip/index.html.md.erb:39]: Maybe link to PIM: https://docs.microsoft.com/en-us/azure/active-directory/privileged-identity-management/pim-configure

File: source/infrastructure/hosting/azure-cip/index.html.md.erb [hunk 3]
```diff
@@ -34,6 +34,34 @@ The new user will receive an invitation by email. Then the service managers can
...
+
+To request access to your eligible assignments, follow the steps below:
+
+* Navigate to the [Azure Portal](https://portal.azure.com/) and login using your `@digital.education.gov.uk` account
```
[the remark this answers is not in the document]
Comment (avinhurry) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb:43]: I think the term `CIP` might be confusing for some people... Could we perhaps say: 

> login using the email you used to register with Azure

What do you think?

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 1 above)
Comment (avinhurry) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb]: Thanks, updated this. I think the test subscription is the staging environment, is this always usually the case?

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 2 above)
Comment (avinhurry) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb]: Done

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (saliceti) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb:43]: This is in the "Azure CIP" section so users should be familiar with it. You can link it to `#onboarding-users` for clarity

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 1 above)
Comment (saliceti) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb]: This is only the case for Teacher services, so I think we should be more generic. Maybe:
>The test subscription (which usually includes staging, preprod, etc. environments), is self approving...

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 3 above)
[the remark this answers is not in the document]
Comment (avinhurry) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb]: Makes perfect sense - have updated

File: source/infrastructure/hosting/azure-cip/index.html.md.erb (hunk 1 above)
Comment (avinhurry) [reply source/infrastructure/hosting/azure-cip/index.html.md.erb]: Have also updated

**Review by saliceti — APPROVED** [review]
👏

**Pull request merged**